content/docs/02.installation/05.kubernetes-gcp-gke.md

```diff
@@ -59,6 +59,29 @@ helm repo add kestra https://helm.kestra.io/
 helm install my-kestra kestra/kestra
 ```
 
+**Workload Identity Setup(Optional)**
+
+If you are using Google Cloud Workload Identity, you can annotate your Kubernetes service account in the Helm chart configuration. This will allow Kestra to automatically use the associated GCP service account for authentication.
```

Comment:
Did you test that this works? if so, can you add a couple of screenshots?

Reply:
I haven't tested it since GKE is not free.

Reply:
no worries